src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -212,4 +212,24 @@ theme.charts = {
   previousPeriod: theme.gray1,
 };
 
+theme.discoverCharts = {
```

Comment:
this should just be the default colors for `theme.charts`

Reply:
So just to clarify - replace current theme.charts with what I have in theme.discoverCharts?

Reply:
yeah